### PR TITLE
Update usage_and_examples.md

### PR DESCRIPTION
**Pull request by cheTesta**
# Summary

Updated a small typo in the documentation page.

# Checklist

- [x] You agree with our [CLA](https://gist.githubusercontent.com/emdgroup-admin/16cc45ea4315c2ef29eb9d9afc36fcf5/raw/abb9c91f15278a62b9ac3e66144bcd27fa9485c2/CLA.md)
- [x] Included tests (or is not applicable).
- [x] Updated [documentation](https://emdgroup.github.io/foundry-dev-tools/) (or is not applicable).
- [x] Used [pre-commit hooks](https://emdgroup.github.io/foundry-dev-tools/develop.html#pre-commit-hooks-formatting) to format and lint the code.

### CODE REVIEW

File: docs/installation.md
Comment (jonas-w) [inline docs/installation.md:1]: this file seems to be edited by accident?

File: docs/installation.md [hunk 1]
```diff
@@ -62,4 +62,4 @@ can be found in the [PySpark Documenation] and the [Spark Documentation].
 [PySpark Documentation]: https://spark.apache.org/docs/latest/api/python/getting_started/install.html
 [Spark Documentation]: https://spark.apache.org/docs/latest/
 [conda environment]: https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html
-[python environment]: https://docs.python.org/3/library/venv.html
\ No newline at end of file
+[python environment]: https://docs.python.org/3/library/venv.html
```
Comment (nicornk) [inline docs/installation.md]: Can you revert your changes to this file?

File: docs/usage_and_examples.md [hunk 2]
```diff
@@ -80,4 +80,4 @@ The FoundryRestClient <FoundryRestClient_usage>
 The fsspec implementation <FoundryFileSystem_usage>
 Configuration <Configuration_usage>
 SSO <SSO_usage>
-```
\ No newline at end of file
+```
```
Comment (nicornk) [inline docs/usage_and_examples.md]: Can you revert the line ending change? If you run the pre-commit hooks this should happen automatically.

File: docs/usage_and_examples.md [hunk 3]
```diff
@@ -51,8 +51,8 @@ for package name and package version:
 ```diff
 -      name=os.environ['PKG_NAME'],
 +      name=os.getenv('PKG_NAME', 'your-package-name'),
--      name=os.environ['PKG_VERSION'],
-+      name=os.getenv('PKG_VERSION', 'your-package-version'),
+-      version=os.environ['PKG_VERSION'],
++      version=os.getenv('PKG_VERSION', 'your-package-version'),
```
Comment (nicornk) [inline docs/usage_and_examples.md:55]: Thanks for the contribution! Good catch.

**Review by nicornk — CHANGES_REQUESTED** [review]
Small changes.

**Review by jonas-w — APPROVED** [review]
Thanks for catching this copy paste error, I modified your fork and removed the accidental changes.

**Review by nicornk — APPROVED** [review]

**Comment by cheTesta** [comment]
Thanks @jonas-w  @nicornk .

**Pull request merged**